lib/msal-node/src/encode/Base64Decode.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export class Base64Decode {
```

Comment:
common?

Reply:
Same, I am using node internal libraries. I am not sure how `browser` can be different. @jasonnutter @pkanher617 to comment.

Reply:
Can we at least have an interface in common?